simplq/src/styles/sidePanel.module.scss

```diff
@@ -3,10 +3,10 @@
 .side-panel {
   padding-top: 1.5rem;
   padding-right: 6rem;
+  max-width: 56vh;
```

Comment:
Why this?

Reply:
removed it. Earlier I used this because the width of the side panel changed when the form is opened. But with 35rem the width still changes so removed it.